packages/medusa/src/services/user.ts

```diff
@@ -244,6 +262,14 @@ class UserService extends TransactionBaseService {
         return Promise.resolve()
       }
 
+      if (this.featureFlagRouter_.isFeatureEnabled(AnalyticsFeatureFlag.key)) {
+        try {
+          await analyticsServiceTx.delete(userId)
+        } catch (error) {
+          console.warn(error)
+        }
```

Comment:
Question: this shouldn't be needed if the delete is idempotent, is there a reason for the warning here (such as missing migrations etc)?